### ISSUE TITLE
Simstrat not running for long time series (>68 years)

### ISSUE THREAD

**Issue by JFeldbauer** [Eawag-AppliedSystemAnalysis/Simstrat#40]
I try to run a single Simulation for about 100 years with daily forcing data and ran into an issue where Simstrat will act as if it successfully ran but actually no output is generated. I tried changing the "MaxLengthInputData" parameter but this did not change anything

I use Simstrat version 2.4 (using the SimstratR R package), an example output of such a run is:

```
[1]` " Simstrat version 2.4"                                                                                                                                             
 [2] " This software has been developed at eawag - Swiss Federal Institute of Aquatic Science and Technology"                                                            
 [3] " "                                                                                                                                                                 
 [4] " [OK] Configuration: simstrat.par"                                                                                                                                 
 [5] " [OK] Grid file successfully read"                                                                                                                                 
 [6] " [OK] Morphology file successfully read"                                                                                                                           
 [7] " [OK] Initial data file successfully read"                                                                                                                         
 [8] " [WARNING] The snow module is turned on. This module needs precipitation data, note that the last column in the forcing file will be interpreted as precipitation."
 [9] " Interval [days]:    1.0000000000000000     "                                                                                                                      
[10] " [OK] Output times successfully read"                                                                                                                              
[11] " [OK] Output depths successfully read"                                                                                                                             
[12] " [OK] Start day: 1.0                 "                                                                                                                             
[13] " [OK] End day: 37254.0             "                                                                                                                               
[14] ""                                                                                                                                                                  
[15] "  -------------------------- "                                                                                                                                     
[16] "    SIMULATION IN PROGRESS   "                                                                                                                                     
[17] "  -------------------------- "                                                                                                                                     
[18] ""                                                                                                                                                                  
[19] "    Time [d]   Surface level [m]       T_surf [degC]     T_bottom [degC]"                                                                                          
[20] ""                                                                                                                                                                  
[21] "  -------------------------- "                                                                                                                                     
[22] "     SIMULATION COMPLETED    "                                                                                                                                     
[23] "  -------------------------- "                                                                                                                                     
[24] ""                                    `
```

But the output file T_out is just the initial temperature profile and the whole run takes less than a second computation time.

```
system.time(SimstratR::run_simstrat("Simstrat/", "simstrat.par"))
   user  system elapsed 
  0.008   0.012   0.020
```

If needed I can provide the model setup to recreate the problem.

All the best,
Johannes

**Comment by Diftin** [comment]
Hi Johannes,

This is a bug and it will be corrected in the next few weeks. Thanks for pointing this out!

**Comment by f-baerenbold** [comment]
This issue is solved in 454eb34.
